docs/configuring-a-resource.md

```diff
@@ -354,24 +357,23 @@ following referencing configuration:
 func Configure(p *config.Provider) {
     p.AddResourceConfigurator("aws_iam_access_key", func (r *config.Resource) {
         r.References["user"] = config.Reference{
-            Type: "User",
+            TerraformName: "aws_iam_user",
         }
     })
 }
 ```
 
-Please note the value of `Type` field needs to be a string representing the Go
-type of the resource. Since, `AccessKey` and `User` resources are under the same
-go package, we don't need to provide the package path. However, this is not
-always the case and referenced resources might be in different package. In that
-case, we would need to provide the full path. Referencing to a [kms key] from
-`aws_ebs_volume` resource is a good example here:
+Please note the value of `TerraformName` field needs to be a string representing
+the name of the Terraform resource. The utilization of the `TerraformName` field
+remains consistent despite the referenced and referencing resources being in
+different groups. Referencing to a [kms key] from `aws_ebs_volume` resource
+is a good example here:
```

Comment:
Suggestion: The following reads more natural to me, but please make sure that I'm not glossing over nuances 🙂

```suggestion
`TerraformName` is the name of the Terraform resource, such as
`aws_iam_user`. Because `TerraformName` uniquely identifies a
Terraform resource, it remains the same when the referenced and
referencing resources are in different groups. A good example is
referencing a [kms key] from `aws_ebs_volume` resource:
```